website/content/en/preview/concepts/deprovisioning.md

```diff
@@ -142,28 +142,25 @@ data:
 
 ## Drift
 
-Karpenter Drift will classify each CRD field as a (1) Static, (2) Dynamic, or (3) Behavioral field and will treat them differently. Static Drift will be a one-way reconciliation, triggered only by CRD changes. Dynamic Drift will be a two-way reconciliation, triggered by machine/node/instance changes and Provisioner or AWSNodetemplate changes.
+Most fields are only triggered by CRD changes. Some special cases will be reconciled two-ways, triggered by machine/node/instance changes and Provisioner or AWSNodeTemplate changes. For one-way reconciliation, values in the CRDs are reflected in the machine in the same way that they’re set. A machine will be detected as drifted if the values in the CRDs do not match the values in the machine. By default, fields are drifted using one-way reconciliation. 
```

Comment:
```suggestion
Drift triggers Karpenter to roll your nodes when the spec of the owning Provisioner changes. Some special cases will be reconciled two-ways, where resolved values are dynamically discovered and changes to the owning Provisioner or NodeTemplate may or may not actually trigger drift on the nodes based on whether the resolved value on the Machine fits within the owning object's dynamically discovered constraints.
```

Some of this feels lengthy. I wonder if we can collapse some of this wording down.